bin/tictactoe

```diff
@@ -1,3 +1,31 @@
 #!/usr/bin/env ruby
-
 require_relative '../config/environment'
+require 'pry'
+
```

Comment:
Prefer to encapsulate CLI logic into a class outside of the bin file. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.